daprdocs/content/en/operations/hosting/kubernetes/kubernetes-upgrade.md

```diff
@@ -14,6 +14,10 @@ description: "Follow these steps to upgrade Dapr on Kubernetes and ensure a smoo
 ## Upgrade existing cluster to {{% dapr-latest-version long="true" %}}
 There are two ways to upgrade the Dapr control plane on a Kubernetes cluster using either the Dapr CLI or Helm.
 
+{{% alert title="Note" color="primary" %}}
```

Comment:
I like this text, however it is best to move this to this section in the docs https://docs.dapr.io/operations/support/support-release-policy/#upgrade-paths that covers upgrades and then putting a reference link to this from this Kubernetes document. 

I would also change 
"guaranteed to be seamless" to "guaranteed tested"
 and 
 "Upgrading more than one minor version at a time is best effort" to "Upgrading more than one minor version at a time is untested and treated as best effort"